update-labels.yml

```
@@ -33,6 +33,7 @@ jobs:
           !contains(github.event.pull_request.labels.*.name, 'waiting for review')
         run: |
           gh pr edit ${{ github.event.pull_request.html_url }} --add-label "waiting for review"
+          echo ${{ github }}
```

dump event context instead
```suggestion
          echo ${{ github.evet }}
```